src/test/java/seedu/address/testutil/RoomBuilder.java

```diff
@@ -34,10 +34,10 @@ public Room build() {
     }
 
     /**
-     * Build a room with name.
+     * Build a room with name and tier.
      */
-    public Room withRoomNum(String roomNum) {
-        return new Room(roomNum);
+    public Room build(String roomNum, Tier tier) {
```

Comment:
Umm, this is not a proper builder pattern. Could you read this https://dzone.com/articles/design-patterns-the-builder-pattern. Just ask me if this is not clear.